apps/web/src/lib/demo/demo.ts

```diff
@@ -16,135 +14,6 @@ export async function isDemoUser() {
     return true;
 }
 
-export function addDeviceCookieStore(cookies: ReadonlyRequestCookies, name: string, category?: string) {
```

Comment:
Wieso löschst du diese ganzen demo Code überall? Ist das alles jetzt über die Versionen abgefangen? Hast du iwie den Demo-User mit Daten in der DB angelegt, der statt der hard-codet Daten angelegt?

Reply:
Mit der transparency Version sind die ganzen Sachen nicht mehr für die Demo relevant, weil die da gar nicht genutzt werden können